lib/repl.js

```diff
@@ -409,7 +408,7 @@ function REPLServer(prompt,
 
     // Check to see if a REPL keyword was used. If it returns true,
     // display next prompt and return.
-    if (cmd && cmd.charAt(0) === '.' && isNaN(parseFloat(cmd))) {
+    if (cmd.charAt(0) === '.' && isNaN(parseFloat(cmd))) {
```

Comment:
I'll double-check, but I don't think it's possible for a non-string to get to this point in the code. I suppose I could make that clear by adding an `assert()`.